src/main.rs

```diff
@@ -921,6 +1005,10 @@ async fn new_job(job: Job) -> Result<impl CancellableJobHandler<LavaUploadableFi
     Ok(Run::new(lava, url, job, cancel_behaviour))
 }
 
+async fn upload_artifact(Path((job, path)): Path<(String, String)>, body: Bytes) {
```

Comment:
This really needs to be a chunked/streaming transfer as the lava job might be quite big files not stuff we want to store in memory perse